src/conn/conn_dhandle.c

```diff
@@ -117,14 +117,14 @@ __conn_dhandle_config_set(WT_SESSION_IMPL *session)
     }
     dhandle->cfg[1] = metaconf;
     dhandle->meta_base = base;
-    dhandle->meta_base_length = base == NULL ? 0 : strlen(base);
-#ifdef HAVE_DIAGNOSTIC
     /*  Save the original metadata value for further check to avoid writing corrupted data. */
-    if (base == NULL)
-        dhandle->orig_meta_base = NULL;
```

Comment:
Why are we taking out this condition? Does this never happen?

Reply:
The dhandle comes initialized so there is no need to reinitialize this field. So I flipped the conditional to only set up actual values when we need them.